spec/requests/users_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  describe 'GET /index' do
+    before(:example) { get(users_path) }
+    it 'check status' do
+      expect(response).to have_http_status(:ok)
+    end
+
+    it 'renders correct template' do
+      expect(response).to render_template('index')
+    end
+
+    it 'renders correct template' do
+      expect(response).to_not render_template('show')
+    end
+
+    it 'check for placeholder text' do
+      expect(response.body).to include('Users#index')
+    end
+
+    it 'check for placeholder text' do
+      expect(response.body).to include('Users#index')
+    end
```

Comment:
- [ ] There is a duplicate test case wriiten here and also on line 46.

Reply:
Done